recipes/moodycamel-concurrentqueue/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(Threads REQUIRED)
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS} Threads::Threads)
```

Comment:
```suggestion
set_property(TARGET example PROPERTY CXX_STANDARD 11)
```

Reply:
You still might want to link to `${CONAN_LIBS}`

Reply:
I've changed this to 

```
target_link_libraries(example ${CONAN_LIBS})
set_property(TARGET example PROPERTY CXX_STANDARD 11)
```